src/features/users/pages/user.js

```diff
@@ -150,28 +132,28 @@ CurrentUserInfo.propTypes = {
 
 const displayName = (user) => (user && user.displayName) || "user"
 
-const NamedCardsList = ({ cards, title, renderEmpty = () => null }) => {
-  if (cards && cards.length !== 0) {
-    return (
-      <>
-        <H1>{title}</H1>
-        <CardsList
-          ids={cards}
-          renderCard={({ card, onUsefulClick }) =>
-            React.createElement(CardItem, {
-              card,
-              key: card.id,
-              onUsefulClick,
-            })
-          }
-        />
-      </>
-    )
-  }
-  return <Col padding="2rem">{renderEmpty()}</Col>
+const NamedCardsList = ({
+  show,
+  isLoading,
+  cards,
+  title,
+  renderEmpty = () => null,
```

Comment:
Duplicated from defaultProps. It is for better static type checking or what?

Reply:
For flowtype checking. Yes